src/components/Input/Input.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import style from "./_input.module.css";
+
+function InputFC(props, ref) {
+	const { placeholder, onChange, type, label } = props;
+
+	if (label && label !== "") {
+		return (
+			<label className={style.label}>
+				<span className={style.label_text}>{label}</span>
+				<br />
+				<input
+					className={style.input}
+					type={type}
+					ref={ref}
+					placeholder={placeholder}
+					onChange={onChange}
+				/>
+			</label>
+		);
+	} else {
+		return (
+			<input
+				className={style.input}
+				type={type}
+				ref={ref}
+				placeholder={placeholder}
+				onChange={onChange}
+			/>
+		);
+	}
```

Comment:
Isto bi moglo da se postogne bez dupliranja input polja.

```
	import { v4 as uuidv4 } from 'uuid';
```

```suggestion
		cosnt inputID = uuidv4();
		return (
			<>
				<label for={inputID } className={style.label}>{label}</label>
			      	<input
				      	id={inputID }
				        className={style.input}
				        type={type}
				        ref={ref}
				        placeholder={placeholder}
				        onChange={onChange}
			      	/>
		      	</>
			
		);
	
```